plugins/org.eclipse.php.composer.api/META-INF/MANIFEST.MF

```diff
@@ -6,8 +6,7 @@ Bundle-SymbolicName: org.eclipse.php.composer.api
 Bundle-Version: 8.3.0.qualifier
 Bundle-Vendor: %Bundle-Vendor
 Bundle-RequiredExecutionEnvironment: JavaSE-17
-Require-Bundle: org.apache.commons.logging,
- org.apache.httpcomponents.httpclient;bundle-version="4.3.6",
+Require-Bundle: org.apache.httpcomponents.httpclient;bundle-version="4.3.6",
  org.apache.httpcomponents.httpcore;bundle-version="4.3.3",
  com.google.gson;bundle-version="2.2.4",
```

Comment:
In general, to future-proof ourself against further changes like this, I think it would be good to use Import-Package for all dependencies that export their packages with a proper version, which is the case for most non-eclipse bundles.